state/backups/restore.go

```diff
@@ -216,10 +216,6 @@ do
 	sed -i.old -r "/^(stateaddresses|apiaddresses):/{
 		n
 		s/- .*(:[0-9]+)/- {{.Address}}\1/
-	}" $agent/agent.conf
-	sed -i.old -r "/^(stateaddresses|apiaddresses):/{
```

Comment:
I tested the sed script interactively and it works with both one and two addresses.